typing/typetexp.ml

```diff
@@ -518,6 +518,9 @@ and transl_type_aux env ~row_context ~aliased ~policy styp =
   | Ptyp_alias(st, alias) ->
       let cty =
         try
+          if not (valid_tyvar_name alias.txt) then
+            raise (Error (alias.loc, env,
+              Invalid_variable_name ("'" ^ alias.txt)));
```

Comment:
There are already too many occurrences of this fishy `"'" ^` business to my liking. Could you factorize the check-and-raise-an-error logic in a `check_tyvar_name env loc name` function?